ui/src/pages/help-center/help-center-card/HelpCenterCard.types.ts

```diff
@@ -0,0 +1,20 @@
+export interface CardButton {
+  text: string;
+  target: string;
```

Comment:
If we are linking inside the app the `target` prop could be omitted (`target?: string`)

This also applies to other props - `url`, `startIconName` etc.

Having them optional should simplify `CARD_DATA` object (omit props which are not required).